cmd/build.go

```diff
@@ -99,7 +101,74 @@ func doBuild(ctx *cli.Context) error {
 	if err != nil {
 		return err
 	}
+	if !args.Config.Userns {
+		binary, err := os.Readlink("/proc/self/exe")
+		if err != nil {
+			return err
+		}
 
+		cmd := []string{
```

Comment:
Here instead of duplicating all the arguments, can we replace os.Args[0] and add `--internal-userns` as Args[1] instead? This way when people add new arguments they won't also have to change this code.

Reply:
Sure, haven't thought about this way. Thanks!